src/Turn.js

```diff
@@ -0,0 +1,32 @@
+class Turn {
+  constructor(guess, card) {
+    this.guess = guess;
+    this.currentCard = card;
+  }
+
+  returnGuess() {
+    return this.guess;
+  }
+
+  returnCard() {
+    return this.currentCard;
+  }
+
+  evaluateGuess() {
+    if (this.currentCard.correctAnswer === this.guess) {
```

Comment:
Is there a cleaner way to write this?  Check back later during a refactor attempt.